docs/work-with-data-apis.md

```diff
@@ -0,0 +1,66 @@
+# Work with unsupported REST Data APIs
+
+HERE Data SDK for C++ supports only a limited number of REST Data APIs. You can find a list of them on our page with the [overall architecture](OverallArchitecture.md).
+
+If you need to work with the REST Data APIs that are currently not supported, you can configure the Data SDK with a unique base URL.
+
+The example below demonstrates how to use the Data SDK to call the API for downloading the difference between two versions of a versioned catalog.
+
+**Example. Download the difference between two versions of a catalog.**
```

Comment:
Do we really need the extra heading?

Reply:
It's not a heading, but just a title/name of the example. I added it to visually and structurally differentiate the example from the other info. As a user, I don't always want to read the whole page when I need just the example.